config/locales/en.yml

```diff
@@ -1,31 +1,16 @@
-# Files in the config/locales directory are used for internationalization and
-# are automatically loaded by Rails. If you want to use locales other than
-# English, add the necessary files in this directory.
-#
-# To use the locales, use `I18n.t`:
-#
-#     I18n.t "hello"
-#
-# In views, this is aliased to just `t`:
-#
-#     <%= t("hello") %>
-#
-# To use a different locale, set it with `I18n.locale`:
-#
-#     I18n.locale = :es
-#
-# This would use the information in config/locales/es.yml.
-#
-# To learn more about the API, please read the Rails Internationalization guide
-# at https://guides.rubyonrails.org/i18n.html.
-#
-# Be aware that YAML interprets the following case-insensitive strings as
-# booleans: `true`, `false`, `on`, `off`, `yes`, `no`. Therefore, these strings
-# must be quoted to be interpreted as strings. For example:
-#
-#     en:
-#       "yes": yup
-#       enabled: "ON"
-
 en:
-  hello: "Hello world"
+  notices:
+    aircraft:
+      created: "Aircraft was successfully created."
+      updated: "Aircraft was successfully updated."
+      destroyed: "Aircraft was successfully destroyed."
+    logbook_entry:
+      created: "Logbook entry was successfully created."
+      updated: "Logbook entry was successfully updated."
+      destroyed: "Logbook entry was successfully destroyed."
+    user:
+      created: "User was successfully created."
+      updated: "User was successfully updated."
+      destroyed: "User was successfully destroyed."
+  mailer:
+    welcome: "Welcome to [DIGITAL FLIGHT SIM]"
```

Comment:
Out of curiosity, are the team's apps multi-lingual?

Reply:
No it is not. The app does have certain sections that supports both English and French text (this is done by having 2 versions of an HTML page, an English and French version).

Though the app as a whole does **not support internationalization**.

Perhaps @jonathanloos may be able to shed light on when (or if) we plan to do such a thing in the future.

Reply:
Yup it is in the books to address, although it does double the work for translation and UI testing so we haven't pulled the trigger on it yet.